packages/cli/src/utils.ts

```diff
@@ -70,16 +70,44 @@ export async function parseConfigFilename(filename: string): Promise<string> {
   return filename;
 }
 
-export function ora(opts?: realOra.Options): realOra.Ora {
-  return realOra({ stream: process.stdout, ...opts });
+class Logger {
+  spinner: realOra.Ora;
```

Comment:
Shall we have the spinner at `TtyLogger`?

Reply:
info, fail, success method still work in non-tty environment. I think spinner should exists in super class

Reply:
IMHO, using `ora` means more interactive in a terminal, otherwise, we need the plain stdout/stderr instead.

Reply:
done